src/main/java/crazypants/enderio/conduit/me/MEConduit.java

```diff
@@ -208,22 +197,12 @@ public boolean canConnectToExternal(ForgeDirection dir, boolean ignoreDisabled)
 
     @Method(modid = "appliedenergistics2")
     private Boolean canConnectToGridNode(IGridNode node, ForgeDirection dir) {
-        if (node.getGridBlock().getConnectableSides().contains(dir.getOpposite())) {
-            if (isDenseUltra()) {
-                return node.hasFlag(GridFlags.ULTRA_DENSE_CAPACITY)
-                        || ((node.hasFlag(GridFlags.DENSE_CAPACITY)) && !node.hasFlag(GridFlags.CANNOT_CARRY));
-            } else if (isDense()) {
-                return true;
-            } else {
-                return !node.hasFlag(GridFlags.ULTRA_DENSE_CAPACITY);
```

Comment:
This is the only somewhat controversial change: The code now no longer checks whether we connect a non-dense ME conduit to a backbone AE2 cable. This was prohibited before but is allowed no.

I have no idea if this is a problem or not. I worked off the assumption that backbone cables are unused so `ULTRA_DENSE_CAPACITY` is never actually set.

Reply:
Shouldn't be a problem, thanks for pointing it out though